elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideResourceConfig.java

```diff
@@ -173,4 +204,15 @@ public static HealthCheckRegistry getHealthCheckRegistry() {
 
         return healthCheckRegistry;
     }
+
+    private Swagger setupSwagger(EntityDictionary dictionary, String path, String title) {
+        Info info = new Info().title(title).version("1.0");
```

Comment:
Elide 5 now interprets the version set in the Swagger document.  If the non-elide models don't have this version - they vanish.  

Don't set a version.